linux/git_airgap_submodule_helper.bsh

```diff
@@ -52,13 +52,26 @@ fi
 #
 # TODO The code to be added to the setup.env should be a git hook
 #
-# NOTE If vsi_common is a sub-submodule, then this function won't work
-# TODO Replace this function with the functionality in the git_clone-airgap-repos
-# just target and git_mirror:update_submodules
+# FIXME If vsi_common is a sub-submodule, then this function won't work
```

Comment:
I think this was a NOTE instead of a a fixme, cause I don't have a way to fix it.